app/routes/administrative-units/administrative-unit/core-data/index.js

```diff
@@ -2,10 +2,16 @@ import Route from '@ember/routing/route';
 
 export default class AdministrativeUnitsAdministrativeUnitCoreDataIndexRoute extends Route {
   model() {
+    const { id } = this.paramsFor('administrative-units.administrative-unit');
     return {
       administrativeUnit: this.modelFor(
         'administrative-units.administrative-unit.core-data'
       ),
+      subOrganizations: this.store.query('organization', {
```

Comment:
I would await this call. At the moment the page will be rendered before the data is loaded.

An alternative is to use a concurrency task and show a loading spinner while the data is loading. Similar to the organizations setup.